signature/policy_eval_signedby.go

```diff
@@ -47,20 +47,22 @@ func (pr *prSignedBy) isSignatureAuthorAccepted(image types.UnparsedImage, sig [
 		return sarRejected, nil, err
 	}
 	defer os.RemoveAll(dir)
-	mech, err := newGPGSigningMechanismInDirectory(dir)
-	if err != nil {
-		return sarRejected, nil, err
-	}
+	/*
+		mech, err := newGPGtypes.SigningMechanismInDirectory(dir)
+		if err != nil {
+			return sarRejected, nil, err
+		}
+	*/
```

Comment:
The choice of "default" mechanism should be on the user of this library (skopeo vs. origin). We should have specific tests that exercise both implementation properly locked in their packages. 

I picked `openpgp` as a default for `signature/` package tests as it does not require any CGO dependency. I'm fine moving some of the test to be package specific as a follow up (even if that means we duplicate tests or create some framework for tests so you can exchange the signing mechanism to prove both implementation produce the same results..)